### PR TITLE
remove dependency on multibuf

### PR DESCRIPTION
**Pull request by klizhentas**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -223,15 +220,10 @@ func (c *Client) GetFile(u string, params url.Values) (*FileResponse, error) {
...
-	defer re.Body.Close()
-	body, err := multibuf.New(re.Body)
-	if err != nil {
-		return nil, err
-	}
 	return &FileResponse{
 		code:    re.StatusCode,
 		headers: re.Header,
-		body:    body,
+		body:    re.Body,
```
Comment (alexlyulkov) [inline client.go]: Now you need to call body.Close() after GetFile(), right?

**Comment by klizhentas** [comment]
yep, that was true for the `multibuf` based implementation too

File: client.go (hunk 1 above)
Comment (klizhentas) [inline client.go]: yep, that was true for the `multibuf` based implementation too

**Comment by alexlyulkov** [comment]
ok, lgtm

**Pull request merged**